core/domain/suggestion_services_test.py

```diff
@@ -780,13 +780,63 @@ def test_update_translation_suggestion_to_change_translation_html(self):
             'content', '<p>old content html</p>').to_dict()
         exploration.states['state 1'].update_content(
             state_domain.SubtitledHtml.from_dict(old_content))
-        exp_services._save_exploration(self.author_id, exploration, '', [])  # pylint: disable=protected-access
+        change_list = [exp_domain.ExplorationChange({
```

Comment:
Why are so many properties needed? Also if this is a common use case, can we make this a utility function in test_utils?

Reply:
I added all the changes by seeing the previously used private mthod i.e. `_save_exploration`. This private method contains all the updates of exploration model. Thanks @nithusha21 I will make this a utility function and commit the changes as requested.

Reply:
I have tested the file after removing the `change_list` and it worked fine. So, I removed them and committed the changes. @nithusha21 PTAL Thank you